exemgr/main.cpp

```diff
@@ -105,6 +105,9 @@ using namespace querytele;
 #include "dbrm.h"
 #endif
 
+
+#include "installdir.h"
+
```

Comment:
doesn't matter functionally, but this can be taken out.  doesn't look like it's used.